web/beacon-app/src/features/projects/components/DeleteProjectModal.tsx

```diff
@@ -0,0 +1,40 @@
+import { Trans } from '@lingui/macro';
+import { Modal } from '@rotational/beacon-core';
+interface DeleteProjectModalProps {
+  close: () => void;
+  isOpen: boolean;
+}
+
+export default function DeleteProjectModal({ close, isOpen }: DeleteProjectModalProps) {
+  return (
+    <Modal
```

Comment:
Has a story been created to fix the issue with the modals?

<img width="471" alt="Screenshot 2023-05-02 at 2 41 24 PM" src="https://user-images.githubusercontent.com/94616884/235756543-b0517a3e-95b7-4eba-8a9c-e93db0c8628c.png">